modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/VersionedRowStore.java

```diff
@@ -534,7 +502,7 @@ private Pair<BinaryRow, BinaryRow> resolve(Value val, Timestamp timestamp) {
      * @return Snapshot future.
      */
     public CompletionStage<Void> snapshot(Path path) {
-        return storage.snapshot(path);
+        throw new UnsupportedOperationException("Snapshots are not supported yet.");
```

Comment:
Do we have ticket for that? Let's add corresponding todo here.

Reply:
"TODO" was added